packages/syncfusion_flutter_datagrid/lib/src/datagrid_widget/widgets/cell_widget.dart

```diff
@@ -697,7 +697,8 @@ class _FilterIcon extends StatelessWidget {
       // the issue by converting the global to local position of the current
       // overlay and used that new offset to display the show menu.
       final RenderBox renderBox =
-          Overlay.of(context).context.findRenderObject()! as RenderBox;
+          Overlay.of(context)?.context.findRenderObject() as RenderBox;
```

Comment:
Check what version of flutter was used here

Reply:
We may not need to change this

Reply:
This throws issue if we dont add condition for this